src/SUMMARY.md

```diff
@@ -6,6 +6,8 @@
 
 - [Installation](installation/installation.md)
   - [System Requirements](installation/system_requirements.md)
+  - [Method 1: Install Epinio and automatically install dependencies](installation/install_epinio_auto.md)
+  - [Method 2: Install Epinio and manually install components](installation/install_epinio_manual.md)
```

Comment:
I am not sure, but I think using the same term (either "dependencies" or "components") is better unless in the second case there are a lot of optional components which are not required and in the automatic case all are required.

Reply:
You are right. Changing to `dependencies`.

Reply:
```suggestion
  - [Method 2: Install Epinio and manually install dependencies](installation/install_epinio_manual.md)
```